babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  presets: ['kcd-scripts/babel', '@babel/preset-typescript'],
```

Comment:
This should not be necessary since there's a `tsconfig.json` to the root of the repo.

https://github.com/kentcdodds/kcd-scripts/blob/50170f0b090593c39f30edc997f62af1c00ecfe1/src/config/babelrc.js#L62

Reply:
I removed it